js/pages/settings.js

```diff
@@ -709,7 +876,7 @@ export const renderContactInformationData = (userData, canWeVoicemailMobile, can
                 <br>
                     <b>
                     <div id="profileHomePhoneNumber">
-                        ${userData[cId.homePhone] ? `${userData[cId.homePhone].substr(0, 3)} - ${userData[cId.homePhone].substr(3, 3)} - ${userData[cId.homePhone].substr(6, 4)}` : ''}
+                        ${optVars.homePhoneNumberComplete ? `${optVars.homePhoneNumberComplete.substr(0, 3)} - ${optVars.homePhoneNumberComplete.substr(3, 3)} - ${optVars.homePhoneNumberComplete.substr(6, 4)}` : ''}
```

Comment:
See comment above.

Reply:
The 'no spaces' option is probably more consistent with the rest of our views across ConnectApp and SMDB. Spaces removed. Updated format: `XXX-XXX-XXXX`